src/Illuminate/Cache/DatabaseStore.php

```diff
@@ -342,6 +378,38 @@ public function flush()
         return true;
     }
 
+    /**
+     * Remove all items from the cache.
+     *
+     * @param  string  $key
+     * @return bool
+     */
+    protected function forgetMany(array $keys)
+    {
+        $this->table()->whereIn('key', array_map(function ($key) {
+            return $this->prefix.$key;
+        }, $keys))->delete();
```

Comment:
if $keys is string pls cast it to array

        $this->table()->whereIn('key', array_map(function ($key) {
            return $this->prefix.$key;
        }, (array)$keys))->delete();

Reply:
Docblock was wrong, and added the types here. Thanks!